src/KeyworkerDashboard/KeyworkerDashboard.js

```diff
@@ -38,6 +38,31 @@ export class KeyworkerDashboard extends Component {
     return { firstDay, lastDay }
   }
 
+  formatChosenDates = (fromDate, toDate) => {
+    const formattedChosenFromDate = moment(fromDate, 'YYYY-MM-DD').format('DD MMMM YYYY')
+    const formattedChosenToDate = moment(toDate, 'YYYY-MM-DD').format('DD MMMM YYYY')
+
+    return { formattedChosenFromDate, formattedChosenToDate }
+  }
+
+  getComparisonDates = (fromDate, toDate) => {
+    // dates come in the format YYYY-MM-DD
+    let from = fromDate
```

Comment:
As a general rule we try to stay away from mutable state and avoid using let. In this situation we'd pull the default date logic into a separate function.